version: "3"
services:
  frontend_server:
    image: hyeeeeejin/condibook-front:0.0.5
    restart: "on-failure"
    ports:
      - 80:80
      - 443:443
    volumes:
      - ./nginx/nginx.conf:/etc/nginx/conf.d/:ro
      - ./certbot/www:/var/www/certbot/:ro
      - ./certbot/conf/:/etc/nginx/ssl/:ro
    networks:
      - condibook

  certbot:
    image: certbot/certbot:latest
    volumes:
      - ./certbot/www/:/var/www/certbot/:rw
      - ./certbot/conf:/etc/letsencrypt/:rw

  backend_server:
    image: hyeeeeejin/condibook-back:0.0.5
    restart: "on-failure"
    ports:
      - 5001:5001
    networks:
      - condibook

  ai_server:
    image: hyeeeeejin/condibook-ai:0.0.5
    restart: "on-failure"
    ports:
      - :5004
    networks:
      - condibook

networks:
  condibook:
